test/test_generated.rb

```diff
@@ -11,14 +11,16 @@ class DataLoadedError < StandardError; end
 
   def generated_without_loading_data(name)
     blob = File.join(samples_path, name)
-    assert_nothing_raised(DataLoadedError, "Data was loaded when calling generated? on #{name}") do
+    begin
       Generated.generated?(blob, lambda { raise DataLoadedError.new })
+    rescue DataLoadedError
+      assert false, "Data was loaded when calling generated? on #{name}"
```

Comment:
This made me realize we should be asserting that `Generated.generated?` returns true too.